FirebaseAuthUI/README.md

```diff
@@ -69,8 +70,9 @@ authUI?.delegate = self
 @import FirebaseAuthUI
 ...
 [FIRApp configure];
-FIRAuthUI *authUI = [FIRAuthUI defaultAuthUI];
-authUI.delegate = self; // Set the delegate to receive callback.
+FIRAuthUI *authUI = [FIRAuthUI default];
```

Comment:
The [changes in the clang importer](https://github.com/apple/swift-evolution/blob/master/proposals/0005-objective-c-name-translation.md#matching) were part of Swift 3. The type at the end of the selector name is deliberately omitted in Swift, since Swift has more complete type information than Objective-C.